### PR TITLE
Add PROXY Protocol support command line parameter document.

### PR DESCRIPTION
**Pull request by blacktear23**
Add PROXY Protocol support command line parameter document.

### CODE REVIEW

**Comment by queenypingcap** [comment]
@shenli PTAL.

**Comment by zimulala** [comment]
PTAL @wentaoxu

**Comment by wentaoxu** [comment]
thanks for your contribution, I think we should check this again after [this](https://github.com/pingcap/tidb/pull/3757)  is merged.

File: op-guide/configuration.md [hunk 1]
```diff
@@ -114,6 +114,18 @@ category: deployment
 + 默认: true
 + 值可以为 (true) 或者 (false). (true) 表明自身会运行 DDL. (false) 表明自身不会运行 DDL
 
+### \-\-proxy-protocol-networks
+
++ PROXY Protocol 允许的代理服务器地址列表，如果需要配置多个地址用`,`分隔。
++ 默认: ""
++ 如果为空，TiDB 会禁用 PROXY Protocol 功能。地址可以使用 IP 地址（192.168.1.50）或者 CIDR （192.168.1.0、24），`*` 代表所有地址。
```
Comment (wentaoxu) [inline op-guide/configuration.md]: `192.168.1.0、24` should be `192.168.1.0/24`?

**Comment by wentaoxu** [comment]
LGTM

**Pull request merged**